apollo-router/src/plugins/rhai/tests.rs

```diff
@@ -579,6 +579,21 @@ fn it_can_create_unix_now() {
     assert!(st <= unix_now && unix_now <= st + 1);
 }
 
+#[test]
+fn it_can_create_unix_ms_now() {
+    let engine = new_rhai_test_engine();
+    let st = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .expect("can get system time")
+        .as_millis() as i64;
+    let unix_ms_now: i64 = engine
+        .eval(r#"unix_ms_now()"#)
+        .expect("can get unix_ms_now() timestamp");
+    // Always difficult to do timing tests. unix_ms_now() should execute within a second of st,
+    // so...
```

Comment:
:)

Perhaps reverse the ordering of the timestamp captures and assert that st >= unix_ms_now ?

Reply:
Actually just followed the same pattern in the test above `it_can_create_unix_now`.
Why does the ordering matter? anyway we will want to check that the second capture ran within a second of the first one, to see that the results makes sense.

Reply:
It's all good and, as you observe the previous test should be re-ordered.

I was basically thinking that if we re-ordered the samples, then we'd just need to assert that the second timestamp was `>=` the first one. That would eliminate the flakiness of guesstimating that it should run within a second.

Anyway, ignore this for now and we'll change it if the test ever becomes flaky.